otter/models/cass.py

```diff
@@ -641,7 +649,12 @@ def wrapper(*args):
             return d
         return wrapper
 
-    def view_manifest(self, with_policies=True, with_webhooks=False):
+    def _group_status(self, status, deleting):
```

Comment:
What's `DISABLED`? Maybe we should just properly support it (perhaps treat it like `DELETING`, but without the the real deletion done by the converger)? Or is it really just some dead status that we don't want to support any more?